docs/tutorials/google/advanced_calibration.ipynb

```diff
@@ -0,0 +1,799 @@
+{
```

Comment:
<p>I think that it might be better to use sampler.run_batch, there is less boiler-plate that way. Is there a particular reason that you're using engine.run_batch?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/4108/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
<p>Agree. I tried moving to <code>sampler.run_batch</code>  but, if I recall correctly, the return type was different and didn't work with <code>to_ground_state_prob</code>.  Just being lazy though, I'll move to <code>sampler</code>  only semantics. </p>

Reply:
<p>All <code>engine</code>s  replaced by <code>sampler</code>s.  <span class="ql-cursor">﻿</span></p>